### ISSUE TITLE
quickstart: openai_gym AttributeError: 'EnvSpec' object has no attribute '_kwargs'. Did you mean: 'kwargs'?

### ISSUE THREAD

**Issue by hpcpony** [tensorforce/tensorforce#863]
tensorforce: 0.6.5
python: 3.10.4

```
[user 0.6.5]$ python3 ../quick.py         
Traceback (most recent call last):
  File "/home/user/0.6.5/../quick.py", line 4, in <module>
    environment = Environment.create(
  File "/opt/RL/Python_3.10.4_tensorforce/lib/python3.10/site-packages/tensorforce/environments/environment.py", line 204, in create
    return Environment.create(
  File "/opt/RL/Python_3.10.4_tensorforce/lib/python3.10/site-packages/tensorforce/environments/environment.py", line 160, in create
    environment = environment(**kwargs)
  File "/opt/RL/Python_3.10.4_tensorforce/lib/python3.10/site-packages/tensorforce/environments/openai_gym.py", line 170, in __init__
    self.environment, self._max_episode_timesteps = self.__class__.create_level(
  File "/opt/RL/Python_3.10.4_tensorforce/lib/python3.10/site-packages/tensorforce/environments/openai_gym.py", line 116, in create_level
    _kwargs = dict(gym.envs.registry.env_specs[level]._kwargs)
AttributeError: 'EnvSpec' object has no attribute '_kwargs'. Did you mean: 'kwargs'?
```
```
[user 0.6.5]$ pip3 show tensorforce
Name: Tensorforce
Version: 0.6.5
Summary: Tensorforce: a TensorFlow library for applied reinforcement learning
Home-page: http://github.com/tensorforce/tensorforce
Author: Alexander Kuhnle
Author-email: tensorforce.team@gmail.com
License: Apache 2.0
Location: /opt/RL/Python_3.10.4_tensorforce/lib/python3.10/site-packages
Requires: gym, h5py, matplotlib, msgpack, msgpack-numpy, numpy, Pillow, tensorflow, tqdm
Required-by: 
```
and the version of gym loaded by `pip3 install ./tensorforce`
```
[user]$ pip3 show gym
Name: gym
Version: 0.23.1
Summary: Gym: A universal API for reinforcement learning environments
Home-page: https://www.gymlibrary.ml/
Author: Gym Community
Author-email: jkterry@umd.edu
License: MIT
Location: /opt/RL/Python_3.10.4_tensorforce/lib/python3.10/site-packages
Requires: cloudpickle, gym-notices, numpy
Required-by: Tensorforce
```

I'm new to tensorforce and not a strong Python programmer, but I don't think this is something that I broke.

**Comment by AlexKuhnle** [comment]
Hi @hpcpony, similar to the other issue, that also seems to be an internal change in the latest Gym versions which require corresponding changes in Tensorforce. Will do, sorry for the trouble. In the meantime you can probably just fix to the [Gym version](https://github.com/tensorforce/tensorforce/blob/master/requirements.txt#L1) that should definitely work, via `pip install --upgrade gym==0.21.0`.

**Comment by hpcpony** [comment]
This fixes # 863 and #864

I started from scratch. Installed Python 3.8.13 (see Note)

Edited the 0.6.5 tensorforce files : setup.py, requirements-all.txt, and requirments.txt (I think this last one is the one that really matters) changing 'gym >= 0.21.0' to 'gym == 0.21.0'.

Did `pip3 install ./tensorforce` and things seemed to install properly.

I can now run the quickstart example without errors.  (I don't get any output, but that's probably another issue...)

Note: I think I saw some dependency version issues (specifically box2d) if you go with Python 3.10.4 so I dropped back to 3.8.13 hoping that it will be closer to expected if/when I start installing environments.
